src/android/ContactAccessorSdk5.java

```diff
@@ -1027,7 +1027,7 @@ else if (accounts.length > 1) {
     private String modifyContact(String id, JSONObject contact, String accountType, String accountName) {
         // Get the RAW_CONTACT_ID which is needed to insert new values in an already existing contact.
         // But not needed to update existing values.
-        int rawId = (Integer.valueOf(getJsonString(contact, "rawId"))).intValue();
+        String rawId = getJsonString(contact, "rawId");
```

Comment:
Should the type be String or long as stated in [the docs](http://developer.android.com/intl/ru/reference/android/provider/ContactsContract.Data.html)? Or does not this matter?